### DISCUSSION TITLE
Using `==` instead of `is`

### DISCUSSION THREAD

**Discussion by StrangeRanger** [IntelliTect/EssentialCSharp.Web#18] [Feedback and Ideas] [answered]
I know Mark Michaelis emphasizes using the `is` or `is not` keywords when comparing against null, instead of the `==` operator, so I thought I'd point out that for the listings on [essentialcsharp.com/delegates-and-lambda-expressions#introduction](https://essentialcsharp.com/delegates-and-lambda-expressions#introduction), `==` is used instead of the `is` keyword. I'm not sure if this happens anywhere else, though.

**Comment by BenjaminMichaelis** [comment, marked as answer]
Thanks for pointing this out! That is correct, that `is` should be used here when comparing against null. This will be updated when the chapter 13 content gets next updated!
